Maple2.Server.Game/GameServer.cs

```diff
@@ -27,10 +26,11 @@ public class GameServer : Server<GameSession> {
     private Dictionary<int, Dictionary<int, ShopItem>> shopItemCache;
     private readonly GameStorage gameStorage;
 
-    public int Channel => Target.GameChannel;
+    private static int _channel = 0;
```

Comment:
style: we don't use `_` prefixes, can just do `this.channel = channel`

Reply:
I can't use this because it's static